lib/streamlit/config.py

```diff
@@ -1211,8 +1220,45 @@ def func_with_lock():
     return disconnect
 
 
+def _check_static_files_config():
+    # Import logger locally to prevent circular references
+    from streamlit.logger import get_logger
+
+    LOGGER = get_logger(__name__)
+
+    # TODO [KAREN] Rephrase warning messages
+    if get_option("server.enableStaticServing"):
+
+        static_folder_path = os.path.abspath("static")
+
+        if not os.path.isdir(static_folder_path):
+            LOGGER.warning(
+                "STATIC FILE SERVING ENABLED, but folder static/ "
+                "doesn't exist :( :( :("
```

Comment:
`Warning: the config option 'server.enableStaticServing' is set to 'true', but the serving folder './static/' doesn't exist. To serve static files, create this folder and put the files in it.`

Reply:
Does this get displayed to the user? . Do we also want to add something like "please be careful about putting any sensitive files in this directory"? 

Another case where this will he helpful is if the user already has a dir called "static" with some sensitive content and we start serving that without a proper notice.